tests/common/mod.rs

```diff
@@ -140,7 +140,7 @@ impl TestEnvironment {
 
     /// Sets up the fake editor to read an edit script from the returned path
     pub fn set_up_fake_editor(&mut self) -> PathBuf {
-        let editor_path = assert_cmd::cargo::cargo_bin("fake-editor");
+        let editor_path = assert_cmd::cargo::cargo_bin("examples/fake-editor");
```

Comment:
Interesting, this really works? How does Cargo know that it needs to build the example before running the test?

Reply:
Yep, it seems that it does (the checks pass). I think you can have regular tests in the files in `examples/`, so maybe `cargo test` builds them for that reason. But that's just my guess.

There's an unstable feature that lets us depend on crates with just binary targets (no library). So once that's in stable, we can switch to that setup. Hopefully this solution doesn't break before then :)

Reply:
Oh, it actually doesn't work :( It works with `cargo nextest run`, however, which is why I didn't notice that it was broken with regular `cargo test`. I don't know why the GitHub actions pass.